lib/linter/linter.js

```diff
@@ -2152,15 +2247,46 @@ class Linter {
          * That means anytime a fix is successfully applied, there will be another pass.
          * Essentially, guaranteeing a minimum of two passes.
          */
+        const slots = internalSlotsMap.get(this);
+
+        // Remove lint times from the last run.
+        if (stats) {
+            delete slots.times;
+        }
+
         do {
             passNumber++;
+            let tTotal;
+
+            if (stats) {
+                tTotal = startTime();
+            }
 
             debug(`Linting code for ${debugTextDescription} (pass ${passNumber})`);
             messages = this.verify(currentText, config, options);
 
             debug(`Generating fixed text for ${debugTextDescription} (pass ${passNumber})`);
+            let t;
+
+            if (stats) {
+                process.hrtime();
+            }
+
             fixedResult = SourceCodeFixer.applyFixes(currentText, messages, shouldFix);
 
+            if (stats) {
+                t = startTime();
+                if (fixedResult.fixed) {
+                    fixPasses++;
+                    const time = endTime(t);
```

Comment:
Looks like `time` doesn't track time spent on fixing the code, perhaps `t = startTime();` should be on line 2272?

Reply:
You are right, the idea is the same (as for *parse*, etc.). We need to start the timer before `SourceCodeFixer.applyFixes()` for the *fix* time to be tracked.